osgp/shared/shared/src/main/java/org/opensmartgridplatform/shared/security/providers/RsaEncryptionProvider.java

```diff
@@ -0,0 +1,58 @@
+package org.opensmartgridplatform.shared.security.providers;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.security.Key;
+import java.security.KeyFactory;
+import java.security.NoSuchAlgorithmException;
+import java.security.spec.AlgorithmParameterSpec;
+import java.security.spec.InvalidKeySpecException;
+import java.security.spec.PKCS8EncodedKeySpec;
+import java.security.spec.X509EncodedKeySpec;
+
+import javax.crypto.Cipher;
+
+import org.opensmartgridplatform.shared.exceptionhandling.EncrypterException;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'org.opensmartgridplatform.shared.exceptionhandling.EncrypterException'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://ci.opensmartgridplatform.org/sonarqube/coding_rules#rule_key=squid%3AUselessImportCheck)